TD1/src/TD1/MergeSort.java

```diff
@@ -0,0 +1,111 @@
+package TD1;
+
+import java.awt.Color;
+import java.awt.Shape;
+import java.lang.management.ManagementFactory;
+import java.lang.management.ThreadMXBean;
+import java.awt.BasicStroke; 
+import org.jfree.chart.ChartPanel; 
+import org.jfree.chart.JFreeChart; 
+import org.jfree.data.xy.XYDataset; 
+import org.jfree.data.xy.XYSeries; 
+import org.jfree.ui.ApplicationFrame; 
+import org.jfree.ui.RefineryUtilities;
+import org.jfree.util.ShapeUtilities;
+import org.jfree.chart.plot.XYPlot; 
+import org.jfree.chart.ChartFactory;
+import org.jfree.chart.plot.Plot;
+import org.jfree.chart.plot.PlotOrientation; 
+import org.jfree.data.xy.XYSeriesCollection;
+import org.jfree.chart.renderer.xy.XYDotRenderer;
+import org.jfree.chart.renderer.xy.XYItemRenderer;
+import org.jfree.chart.renderer.xy.XYLineAndShapeRenderer;
+
+public class MergeSort extends ApplicationFrame 

```

Comment:
The serializable class MergeSort does not declare a static final serialVersionUID field of type 
 long